styles/components/dropdown-tree-end.css

```diff
@@ -0,0 +1,12 @@
+@layer component {
+  .dropdown-tree-end {
+    box-shadow: 2px 2px 0px 0px #000000;
```

Comment:
Is not the same as `shadow-rb-black`?

Reply:
This is a different class since the other one handles responsiveness. I added a new utility that already existed in another file to reuse it for this type of element.